yt/utilities/lodgeit.py

```diff
@@ -326,7 +332,3 @@ def main(
         open_webbrowser(url)
     if clipboard:
         copy_url(url)
-
-
-if __name__ == "__main__":
-    sys.exit(main())
```

Comment:
note: it's been broken for 11 years and no one complained so I take it that no one will miss it.

Reply:
I think it was usually called by a different wrapper function.

Reply:
yeah it's integrated in yt's CLI